controllers/actions.github.com/autoscalingrunnerset_controller.go

```diff
@@ -113,6 +116,16 @@ func (r *AutoscalingRunnerSetReconciler) Reconcile(ctx context.Context, req ctrl
 			return ctrl.Result{}, err
 		}
 
+		requeue, err := r.cleanupPermissions(ctx, autoscalingRunnerSet, log)
```

Comment:
I thought to be more explicit that we need to requeue
Done is signalling if we should reconcile again, while requeue is signalling that we need to requeue the same request.

Since the behaviour is different, one would require explicit requeue while another will wait for update to trigger reconcile, I thought to separate it to be more explicit